src/gobapi/__main__.py

```diff
@@ -6,8 +6,20 @@
 
 """
 import os
-
+from uuid import uuid4
+from flask import request
 from gobapi.api import get_app
 
 app = get_app()
+
+
+@app.before_request
+def set_request_id():
```

Comment:
This breaks tests because there is no app context. An app-fixture would help a lot, also reduces the need of mocks.
https://flask.palletsprojects.com/en/1.1.x/testing/